witness/golang/omniwitness/monolith/README.md

```diff
@@ -6,7 +6,7 @@ common configuration steps such as key generation and creating github credential
 This version uses a single `main` construct to invoke all of the sub-components needed to
 implement the OmniWitness. 
 
-## Running
+## Running Bare
```

Comment:
How about "locally" or "... the binary"?
I wouldn't know what "Bare" meant in this context until I'd read through the section

Reply:
I renamed it to "running without docker" and moved it to after the docker section. I think that makes more sense?